packages/components/src/components/hint-box/hint-box.tsx

```diff
@@ -1,9 +1,17 @@
-import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
+import classNames from 'classnames';
 import Icon from '../icon/icon';
 
-const HintBox = ({ className, icon, is_info, is_warn, is_danger, message }) => {
+type THintBox = {
+    className?: string;
+    icon: string;
+    is_info?: boolean;
+    is_warn?: boolean;
+    is_danger?: boolean;
+    message: string | React.ReactElement;
```

Comment:
React.ReactElement itself includes string | JSX element. So we can omit string

Reply:
@likhith-deriv I'm not sure. ReactElement is any JSX element, and it can be <>'some string</>, but not string.
ReactNode includes string and everything else